### DISCUSSION TITLE
Messages do not arrive on WhatsApp

### DISCUSSION THREAD

**Discussion by shukri0** [AntlerVC/firetable#1414] [Support / Q&A] [answered]
mrjsh0kri@gmail.com
Project: sambot-5rpu10
I am suffering from the problem of messages not arriving on WhatsApp
Although it is on the website and the response is from GPT chat
But there is something wrong
I tried deleting the first, second and third projects to no avail

**Comment by Chadha93** [comment, marked as answer]
Hi, @shukri0 Could you please elaborate on the use case? 

If you are creating a WhatsApp Bot to receive messages, you can do it using [BuildShip](https://buildship.com/) easily.

Check this demo on[ AI WhatsApp Bot - Visual Workflow Builder, low-code way](https://www.youtube.com/watch?v=QyR8Sautvhg).


I invite you to join our [Discord](https://www.rowy.io/discord) and post this on the community support channel. 

--
Cheers,
Gaurav Chadha | Technical Support Engineer | [Rowy](https://rowy.app/) + [BuildShip](https://buildship.com/)
backend for everyone
